### PR TITLE
Update faqs

### LINKED ISSUE #80: Suggested updates in FAQs
Hi @pythonindia/pycon-2018-team 
Below are some of the changes I think will be good to update on the FAQ page of the website: 
1. Q. Where can I find updates related to PyCon India 2018?
We can remove the IRC channel & put twitter there. 

2. Q. I want to present a talk/workshop at the conference. What should I do?
We can add the link to the CFP page now.

4. Q. I want to volunteer for the conference. What should I do?
I think we can point it to the email/blog post with the form for volunteer registration.

9. Q. Where can I stay while in Hyderabad for the conference?
I think we can mention we will be listing down some recommended hotels near the venue, possibly with discounts.

### PR DESCRIPTION
**Pull request by ananyo2012**
Fixes #80 
Updated the FAQs. @pythonindia/pycon-2018-team please review.

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -147,7 +147,7 @@ <h5><b>Q. What are the workshop, Developer Sprints and conference timings?</b></
           <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
           <ul class="faq-answer">
                 <li>- Full conference pass for two days.</li>
-                <li>- Lunch and Hi-Tea</li>
+                <li>- Lunch and High-Tea</li>
```
Comment (vipulgupta2048) [inline faq.html]: - Wouldn't access to Developer Sprints also be there. Also, what about workshops. We might need to provide complete information here. 
- We need this to sound and look attractive, Hence I suggest. 
A full conference pass for two days that includes access to all keynotes, talks, sessions, open-spaces and much more.

**Comment by ananyo2012** [comment]
On Wed 23 May, 2018, 10:00 AM Vipul Gupta, <notifications@github.com> wrote:

> *@vipulgupta2048* commented on this pull request.
> ------------------------------
>
> In faq.html
> <https://github.com/pythonindia/inpycon2018/pull/93#discussion_r190119985>
> :
>
> > @@ -147,7 +147,7 @@ <h5><b>Q. What are the workshop, Developer Sprints and conference timings?</b></
>            <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
>            <ul class="faq-answer">
>                  <li>- Full conference pass for two days.</li>
> -                <li>- Lunch and Hi-Tea</li>
> +                <li>- Lunch and High-Tea</li>
>
>
>    - Wouldn't access to Developer Sprints also be there. Also, what about
>    workshops. We might need to provide complete information here.
>
> We still didn't finalize how we will be managing tickets for dev sprints.
We will put that information once it is finalized. This is only for
conference. Workshop ticket will be different. We will cover that detail
later as well.

>
>    - We need this to sound and look attractive, Hence I suggest.
>    A full conference pass for two days that includes access to all
>    keynotes, talks, sessions, open-spaces and much more
>
> Noted. Will update it. But we didn't finalize on open spaces, so I will
update that once it is finalized

>

**Comment by ananyo2012** [comment]
Fixed. Anymore change needed ?

**Comment by ananyo2012** [comment]
Merging this PR

**Pull request merged**